mate-panel/drawer.c

```diff
@@ -587,7 +587,10 @@ panel_drawer_prepare (const char  *drawer_id,
     g_settings_set_boolean (settings, PANEL_OBJECT_USE_CUSTOM_ICON_KEY, use_custom_icon);
 
     if (custom_icon) {
-        g_settings_set_string (settings, PANEL_OBJECT_CUSTOM_ICON_KEY, custom_icon);
```

Comment:
Are these drawer changes necessary?

Reply:
Yes, because matemenu_tree_directory_get_icon() now return GIcon object, not the icon_name string.

Reply:
It feels wrong, now icon is not saved and most likely is / will be regression. Or if saving icon in settings has never been needed then I would suggest to remove it in separate commit before this API change...

Reply:
Oh, I guess I missed something... I saw that it removes line, somehow missed that new code is added, sorry for that.

Reply:
It's ok :slightly_smiling_face: